src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>112.0.1</Version>
-    <PackageReleaseNotes>Extended the lookup to determine the underlying sql type for DateOnly</PackageReleaseNotes>
+    <Version>112.0.2</Version>
```

```suggestion
    <Version>113.0.0</Version>
```
It is clearly no bugfix, but a backwards incompatibele change